src/std/db/conpool.ss

```diff
@@ -60,7 +60,7 @@
     (mutex-lock! cp.mx)
     (set! cp.out (remq conn cp.out))
     (mutex-unlock! cp.mx)
-    {destroy conn}))
+    {conn.destroy}))
```

Comment:
How does that work over `{}` ?

Reply:
I added support for dots in the @method macro in the ptelude in the original using pr.